k9mail-library/src/main/java/com/fsck/k9/mail/Address.java

```diff
@@ -95,7 +95,8 @@ public String getPersonal() {
         return mPersonal;
     }
 
-    public void setPersonal(String personal) {
+    public void setPersonal(String newPersonal) {
+        String personal = newPersonal;
```

Comment:
There's a checkstyle warning for assigning parameters.

        <module name="ParameterAssignment">
            <property name="severity" value="warning"/>
        </module>

I've been slowly trying to get rid of the lint, checkstyle and findBugs warnings. But this one is a bit lame so we could just turn it off. 

Good catch about the trim point.